etc/requirements.in

```diff
@@ -0,0 +1,63 @@
+# Incompatible with earlier PIP versions
```

Comment:
Do we still actually need pip and setuptools here?

Reply:
I looked into this and removed them in #2610: they are not necessary. (That PR is failing for unrelated reasons, which this one should help to fix.)

+1 on just leaving them here for now though.